odoo/sql_db.py

```diff
@@ -60,12 +60,14 @@ def adapt_string(adapted):
 from inspect import currentframe
 
 
-def flush_env(cr):
+def flush_env(cr, clear=True):
```

Comment:
Might be better to not have a default value here. And it should probably be a keyword-only parameter either way.

Reply:
This is a breaking API change, and we are on a stable version here.  So the default is necessary.